scripts/nginx/prowlarr.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# Nginx conf for Radarr v3
+# Flying sausages 2020
+# Refactored by Bakerboy448 2021
+master=$(cut -d: -f1 < /root/.master.info)
+app_name="prowlarr"
+if ! PROWLARR_OWNER="$(swizdb get $app_name/owner)"; then
+    PROWLARR_OWNER=$(_get_master_username)
+else
+    PROWLARR_OWNER="$(swizdb get $app_name/owner)"
+fi
+app_port="9696"
+user="$PROWLARR_OWNER"
+app_servicename="${app_name}"
+app_configdir="/home/$user/.config/${app_name^}"
+app_baseurl=$app_name
+
+cat > /etc/nginx/apps/$app_name.conf << PROWLARR
+location /$app_name {
```

Comment:
I think you want `$app_baseurl`, I'd argue about just getting rid of this var as mentioned above.